tests/Support/Database/Seeders/PermissionSeeder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace OwowAgency\Gossip\Tests\Support\Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Spatie\Permission\Models\Permission;
+use Spatie\Permission\Models\Role;
+
+class PermissionSeeder extends Seeder
```

Comment:
This is only being implemented for test, however it is important to have in the actual application for it to work.

Reply:
I've added a note in the documentation that I need to update this. I think this differs for every application so the developer needs to do something to be able to have the correct permissions. 

So conclusion: this will be fixed in a way somewhere else.